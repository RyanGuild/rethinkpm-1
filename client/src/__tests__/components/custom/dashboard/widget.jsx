import Widget from '../../../../components/custom/dashboard/Widget';
import React from 'react';
import '@testing-library/jest-dom/extend-expect';
import {render} from '@testing-library/react';

test('reander and snapshot', () => {
  let {container} = render(<Widget />);
  expect(container).not.toBeNull();
  expect(container).toMatchSnapshot();
});
