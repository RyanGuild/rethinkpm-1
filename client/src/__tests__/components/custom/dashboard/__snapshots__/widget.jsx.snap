// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`reander and snapshot 1`] = `
<div>
  <div
    class="widget bg-light"
  >
    <div>
      <h2>
        Name Of Module
      </h2>
      <p>
        KPI Name : KPI Result
      </p>
      <div
        class="my-1 module-body"
      >
        <p>
          module body
        </p>
      </div>
    </div>
  </div>
</div>
`;
